### ISSUE TITLE
Documentation: broken links for samples, sample configs

### ISSUE THREAD

**Issue by teaganryley** [fgpv-vpgf/fgpv-vpgf#4070]
Reproduction:

1. User navigates to: Map Author > Introduction > [Helpful Links](https://fgpv-vpgf.github.io/fgpv-vpgf/master/docs/#/mapauthor/intro?id=helpful-links)
2. User clicks on [Samples](http://fgpv.cloudapp.net/demo/master/prod/samples/index-samples.html) OR
3. User clicks on [Configs for those samples](http://fgpv.cloudapp.net/demo/master/prod/samples/config/)
4. Connections refused on redirect

**Comment by james-rae** [comment]
Yep server died, we were hoping we could get it back and dragging our feet, but looks like it's toast.

You can find a current sample site [here](https://fgpv-vpgf.github.io/fgpv-vpgf/master/samples/index-samples.html), there is no nice "table of contents" or directory pages at the moment.

Team: either re-direct the links (might need to rename stuff since we can only target singular pages for now), or remove the links. The "configs" link can probably just direct to the [repo folder](https://github.com/fgpv-vpgf/fgpv-vpgf/tree/master/packages/ramp-core/src/content/samples/config)

**Comment by teaganryley** [comment]
Much appreciated!
